horde/database/functions.py

```diff
@@ -265,7 +265,7 @@ def worker_exists(worker_id):
     return wc
 
 
-def get_available_models(filter_model_name: str = None):
+def get_available_models(filter_model_name: str = None) -> list[dict]:

```

Comment:
Horde is still largely running in Python 3.9 so typing is not well supported there. I can't remember if this declaration works in 3.9